src/main/java/com/tarasenko/sorting_algorithms/CombSort.java

```diff
@@ -0,0 +1,42 @@
+package com.tarasenko.sorting_algorithms;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class CombSort {
+
+    public void sort(int[] array) {
+        int gap = array.length;
+        boolean swapped = true;
+        while (gap > 1 || swapped) {
+            if (gap > 1) {
+                gap = (int) (gap / 1.247330950103979);
+            }
```

Comment:
It`s better to use static variable with magic numbers

Reply:
Done